src/include/txn.h

```diff
@@ -41,7 +41,10 @@
 #define	WT_SESSION_IS_CHECKPOINT(s)					\
 	((s)->id != 0 && (s)->id == S2C(s)->txn_global.checkpoint_id)
 
-#define	WT_TS_NONE	0		/* No timestamp */
+#define	WT_TS_NONE	0		/* No (or earliest) timestamp) */
+#define	WT_TS_FIXME	37		/* Fake timestamp */
```

Comment:
Would you be OK with merging with this in here? It seems OK to me, but is a little odd so I thought I'd seek confirmation.

Reply:
I agree it's odd.

We don't yet use any of these timestamps, and this is literally a flag meaning "this timestamp isn't right and needs to be fixed before it's used for real", maybe there's a better name?